activestorage/test/template/image_tag_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+require "database/setup"
+
+class User < ActiveRecord::Base
+  has_one_attached :avatar
+end
+
+class ActiveStorage::ImageTagTest < ActionView::TestCase
```

Comment:
I'm more inclined with `test/activestorage` since we may change the implementation in the future to not use Active Record (not that I see this happening but better to be safe)